2-mandatory/7-eligible-students.js

```diff
@@ -14,10 +14,18 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
  */
 
-function eligibleStudents(attendances) {}
+function eligibleStudents(attendances) {
```

Comment:
This works! A more simple way could be to filter then map

Reply:
Map is still a little bit confusing. Sometimes I get and sometimes I don't.